## cancels a trajectory(ies)

# if 0, cancels all trajectories, otherwise cancels the trajectory with the specified id
# if no such trajectory exists return fail
uint32 trajectoryid 
---
